### PR TITLE
ember-cli-update

### PR DESCRIPTION
**Pull request by alexdiliberto**
With the [fix](https://github.com/RSSchermer/ember-spectrum-color-picker/commit/832469c7eec578f2389829eebc2059fe4ea5836a) for https://github.com/RSSchermer/ember-spectrum-color-picker/issues/30, I was still seeing some warnings because it was still attaching to the global Ember object like this

```
DEPRECATION: Using Ember.$() has been deprecated, use `import jQuery from 'jquery';` instead
```

```js
initSpectrum(opts) {
  // built dist compiled like this
  Ember.$(this.element).spectrum(opts);
}
```

This PR:
* gets travis green again
* updates to cli v3.12
* remove all the deprecations
* includes a breaking change of node v4.0 -> v8.0 base version support which would require a major Semver NPM release for this addon
* updates many dependencies

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,26 @@
-# Ember-spectrum-color-picker
+my-addon
```
Comment (RSSchermer) [inline README.md]: The main readme header seems to have gotten overwritten with the generator default

**Comment by RSSchermer** [comment]
Hey, thank you very much!

Looks mostly good to me, except for the line tagged with the comment above.

File: README.md (hunk 1 above)
Comment (alexdiliberto) [reply README.md]: good catch, i'll fix

**Comment by alexdiliberto** [comment]
@RSSchermer Should be good to review again, let me know if you spot anything else. Thanks

**Comment by RSSchermer** [comment]
My sincere apologies, this had totally slipped my mind.

Merged and released as 0.10.0, thanks again!

**Pull request merged**